### DISCUSSION TITLE
How to get rabbitmq_delayed_message_exchange-20171215-3.6.x.zip ?

### DISCUSSION THREAD

**Discussion by charnet1019** [rabbitmq/rabbitmq-delayed-message-exchange#160] [General]
Rabbitmq Version: 3.6.14

Forbidden to access https://dl.bintray.com/rabbitmq/community-plugins/3.6.x/rabbitmq_delayed_message_exchange/rabbitmq_delayed_message_exchange-20171215-3.6.x.zip, 
are there other download channels?

Thanks

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
[Bintray has shut down on May 1st](https://blog.rabbitmq.com/posts/2021/03/migrate-off-of-bintray/).

There is no alternative archive of 3.6 community plugin releases and we are not interested in spending any time on it as 3.6 [has been out of support for 3 years](https://www.rabbitmq.com/versions.html).

You can always build the plugin from source using a relevant GitHub tag.

**Comment by michaelklishin** [comment]
By the way, supported plugin series (3.8.x) are [distributed via GitHub releases](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases).

The only option we can recommend is [upgrading to 3.8.16](https://www.rabbitmq.com/upgrade.html) as you are close to [40 releases behind](https://www.rabbitmq.com/changelog.html).

**Reply by charnet1019** [reply]
Ok, thank you.

**Comment by ykfq** [comment]
Here is the origin file I downloaded last year

[rabbitmq_delayed_message_exchange-20171215-3.6.x.zip](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/files/6739267/rabbitmq_delayed_message_exchange-20171215-3.6.x.zip)

**Reply by QjjQ** [reply]
Thank you ,I find it for a long long long time

**Reply by OneCodeToEnd** [reply]
Thank you ,I find it for a long long long time

**Reply by corthay** [reply]
Thank you ,I find it for a long long long time
